### PR TITLE
CODEX Post R39 omnibus bug fixes

### PR DESCRIPTION
**Pull request by mjallison42**
Bug fixes for CODEX, recover from R39 regressions, etc. Primarily answer isse #267, "CODEX is broken"

### CODE REVIEW

File: codex/inc/x16_kernal.inc [hunk 1]
```diff
@@ -108,17 +115,33 @@
 	FILE_LA       = $028f
 	FILE_SA       = $0290
 	FILE_FA       = $0291
+   cscrmd        = $0261
```
Comment (mist64) [inline codex/inc/x16_kernal.inc:118]: you can call `screen_mode` with .C=1 to get this.

In general, these KERNAL addresses will keep breaking, is there a way to do without them? What APIs do you need?

File: kernal/drivers/x16/screen.s [hunk 2]
```diff
@@ -26,6 +26,9 @@
 ; for monitor
 .export pnt
 
+; for codex
+.export cscrmd
```
Comment (mist64) [inline kernal/drivers/x16/screen.s:30]: This is not actually used, `cscrmd` is hard-coded in CodeX.

**Review by mist64 — CHANGES_REQUESTED** [review]

File: codex/inc/x16_kernal.inc (hunk 1 above)
Comment (mjallison42) [reply codex/inc/x16_kernal.inc]: Good code review! This is an old hanger on, switched to proper JSRFAR call. (docs need to be updated). Also remove LA/SA/FA, I need to reconsider how to preserve state during BRK handling. Any way to get current LA/SA/FA?
PR has been updated to remove these references.

File: kernal/drivers/x16/screen.s (hunk 2 above)
Comment (mjallison42) [reply kernal/drivers/x16/screen.s]: Removed, thanks for keeping me honest.

**Pull request merged**